### PR TITLE
Switch - refactor driver to use new implementation

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
Switch - refactor driver to use new implementation

## Changelog
Switch - refactor driver to use new implementation

### CODE REVIEW

**Comment by ethanshar** [comment]
@Inbal-Tish Is this ready for review?

**Comment by Inbal-Tish** [comment]
> @Inbal-Tish Is this ready for review?

Yes

File: src/components/switch/switch.driver.ts [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
-  isDisabled = async () => (await this.getElementProps()).accessibilityState?.disabled === true;
-
-  isChecked = async () => (await this.getElementProps()).accessibilityValue?.text === '1';
-
-  // @ts-ignore
-  getColor = async () => (await this.getElementProps()).style.backgroundColor;
-}
+export const SwitchDriver = (props: ComponentProps) => {
+  const driver = usePressableDriver<SwitchProps>(useComponentDriver(props));
+  console.log('INBAL - SwitchDriver:', driver.getProps());
```
Comment (adids1221) [inline src/components/switch/switch.driver.ts:8]: Let's remove this log

**Review by adids1221 — COMMENTED** [review]
I think you missed this log, please solve the `testKit` conflict

File: src/components/switch/switch.driver.ts (hunk 1 above)
Comment (Inbal-Tish) [inline src/components/switch/switch.driver.ts:8]:
```suggestion
```

Comment (adids1221) [inline src/components/switch/switch.driver.ts:8]:
```suggestion

```

**Review by adids1221 — APPROVED** [review]

**Pull request merged**